pkgs/servers/monitoring/prometheus/rtl_433-exporter.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub, pkgs }:
+
+buildGoModule rec {
+  name = "rtl_433-exporter";
```

Comment:
```suggestion
  pname = "rtl_433-exporter";
```

Reply:
Maybe even rename the file and set pname to `prometheus-rtl_433-exporter`, since it is later referred to as that (starting with `all-packages.nix`). But as long as it is consistent in the end, I am fine with the current name as well :)

Reply:
Looking at neighboring files, they all use `pname` without the `prometheus-` suffix. Keeping it that way for consistency. Please reopen if you feel strongly about it :-)

Reply:
Oh right, that's fine :+1: